tests/test_integration_association.py

```diff
@@ -893,6 +942,82 @@ def test_edit_association_no_overwrite(self):
                 expected,
             )
 
+    def test_edit_association_fails_validation(self):
+        """Only one active association is allowed for each employee in each
+        org unit """
+        self.load_sample_structures()
+
+        # Check the POST request
+        userid = "53181ed2-f1de-4c4a-a8fd-ab358c2c454a"
+        unitid = "da77153e-30f3-4dc2-a611-ee912a28d8aa"
+        association_uuid = 'c2153d5d-4a2b-492d-a18c-c498f7bb6221'
+
+        payload = [
+            {
+                "type": "association",
+                "org_unit": {'uuid': unitid},
+                "job_function": {
+                    'uuid': "3ef81e52-0deb-487d-9d0e-a69bbe0277d8"},
+                "association_type": {
+                    'uuid': "62ec821f-4179-4758-bfdf-134529d186e9"
+                },
+                "address": {
+                    'address_type': {
+                        'example': '20304060',
+                        'name': 'Telefonnummer',
+                        'scope': 'PHONE',
+                        'user_key': 'Telefon',
+                        'uuid': '1d1d3711-5af4-4084-99b3-df2b8752fdec',
+                    },
+                    'value': '33369696',
+                },
+                "validity": {
+                    "from": "2017-12-01T00:00:00+01",
+                    "to": "2017-12-02T00:00:00+01",
+                },
+            }
+        ]
+
+        self._perform_request('/service/e/{}/create'.format(userid),
+                              json=payload)
```

Comment:
Shouldn't we check the result of this invocation? Alternatively, we should perhaps change the result be useful here.